swap_meet/vendor.py

```diff
@@ -1,2 +1,74 @@
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+        
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self,category):
+        list_of_items = []
+        for item in self.inventory:
+            if category ==item.category:
```

Comment:
Stay consistent with your syntax.

```suggestion
            if category == item.category:
```